### ISSUE TITLE
dockerize integration tests for memcached, redis and postgres

### ISSUE THREAD

**Issue by adriancole** [openzipkin/zipkin-js#402]
Right now, developers on all platforms and also CI needs to be aware of memcached, redis and postgres. They also need to be aware of docker, which is used to manage kafka. I'd suggest using docker for managing memcached, redis and postgres so folks only have one thing to be concerned with.

See #401

**Comment by eirslett** [comment]
Relevant: https://github.com/testcontainers/testcontainers-node

**Comment by adriancole** [comment]
yep good point @jcchavezs and @eirslett, we've had good luck with testcontainers in java and it has a wait strategy we can use for containers listen ports
